packages/react/src/transactions/adapters/responders/FollowProfileResponder.tsx

```diff
@@ -0,0 +1,41 @@
+import { TriStateValue } from '@lens-protocol/api-bindings';
+import { FollowRequest } from '@lens-protocol/domain/use-cases/profile';
+import {
+  ITransactionResponder,
+  TransactionData,
+} from '@lens-protocol/domain/use-cases/transactions';
+
+import { IProfileCacheManager } from '../../../profile/adapters/IProfileCacheManager';
+
+export class FollowProfileResponder implements ITransactionResponder<FollowRequest> {
+  constructor(private readonly profileCacheManager: IProfileCacheManager) {}
+
+  async prepare({ request }: TransactionData<FollowRequest>) {
+    this.profileCacheManager.updateProfile(request.profileId, (current) => {
+      return {
+        ...current,
+        stats: {
+          ...current.stats,
+          followers: current.stats.followers + 1,
+        },
+        operations: {
+          ...current.operations,
+          canFollow: TriStateValue.No,
+          isFollowedByMe: {
+            __typename: 'OptimisticStatusResult',
+            value: true,
+            isFinalisedOnchain: false,
+          },
+        },
+      };
+    });
```

Comment:
These should be overidden via field policies to achieve a reliable result on refresh or other tabs.

Reply:
see https://github.com/lens-protocol/lens-sdk/pull/611